### ISSUE TITLE
remove format-query-block rule

### ISSUE THREAD

**Issue by tyankatsu0105** [gridsome/eslint-plugin-gridsome#89]
**Please write rule name what you want to improve:**
format-query-block

**How are the rules improved?:**
Prettier will supports custom block since v2.1.0
https://github.com/prettier/prettier/pull/8023

So, this rule not necessary anymore.

**Provide 2-3 code examples that this rule should warn about:**

```vue

```

**Additional context**

**Comment by tyankatsu0105** [comment]
Prettier supports custom block format🎉
https://prettier.io/blog/2020/08/24/2.1.0.html#vue
